project/src/main/java/Register.java

```diff
@@ -0,0 +1,46 @@
+import java.io.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+import java.sql.*;
+
+public class Register extends HttpServlet {
+	
+	protected void doPost(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+		response.setContentType("text/html;charset=UTF-8");
+		PrintWriter out = response.getWriter();
+	
+		String First_name = request.getParameter("First_name");
+		String Last_name = request.getParameter("Last_name");
+        String Username = request.getParameter("Username");
+		String Pass = request.getParameter("Pass");

```

Comment:
I think these should be variables of the Register class so they can be used in other methods/functions.